index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-setup-and-mobile-first</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="menu">
+        <a href="#">My Logo</a>
+        <div class="menu-bar">
+          <div id="menu-bar-1"></div>
+          <div id="menu-bar-2"></div>
+          <div id="menu-bar-3"></div>
+        </div>
+      </nav>
+      <section class="main-section">
+        <div class="content">
+          <div class="heading">
+            <h2>I’m Muhammed Glad to see you!</h2>
+          </div>
+          <div class="heading-text">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="social-media">
+            <p>LET’S CONNECT</p>
+            <div class="icons">
+              <img src="/assets/Vector.svg" alt="twitter" />
+              <img src="/assets/Linkedin icon.svg" alt="linkedin" />
+              <img src="/assets/Vector (2).svg" alt="m" />
+              <img src="/assets/Vector (1).svg" alt="github" />
+              <img src="/assets/Vector (3).svg" alt="s" />
+            </div>
```

Comment:
- Kindly, your social list of icons has to be a  `ul` tag to group them, and a `li` per icon, for sematic reasons 👍👍